homeworks/rostyslav.havuka_RostyslavHavuka/6-typeScript/src/main.css

```diff
@@ -0,0 +1,53 @@
+.red-square {
+  background-color: #EE0606;
+  position: fixed;
+  right: 49%;
+  top: 33%;
+}
+
+.key {
+  position: fixed;
+  right: 41.8%;
+  top: 34.7%;
+  font-size: 40px;
+  color: #020202
+}
+
+span[data-cube-score] {
```

Comment:
Same here. You didn't fix the previous comment.
Don't use data-attribute selectors in CSS. Review the whole file to fix this error